experiments/models.py

```diff
@@ -95,7 +95,12 @@ def is_open(self):
         """An experiment is open if it is both approved and set to 'open'.
         While an experiment should not be able to have the status 'open' without being approved,
         we check both to be sure."""
-        return self.state == self.OPEN and self.approved
+        experiment_open = self.state == self.OPEN and self.approved
+        groups_open = True
+        if self.uses_groups():
+            groups_open = any((group.is_open() for group in self.targetgroup_set.all()))
```

Comment:
It might be an idea to refactor this into a ``has_open_groups`` method? I could live without tho ;)

Reply:
I started doing that, but then it seems counterintuitive when `uses_groups()` is false, so I'd rather not have that for now.

Reply:
Fair enough